tensorflow/lite/kernels/internal/runtime_shape.h

```diff
@@ -248,7 +248,11 @@ inline int Offset(const RuntimeShape& shape, int i0, int i1, int i2, int i3,
 }
 
 inline int Offset(const RuntimeShape& shape, int* index) {
-  return Offset(shape, index[0], index[1], index[2], index[3]);
+  if (shape.DimensionsCount() == 5) {
```

Comment:
This shouldn't be needed anymore once Offset has been removed from the loop above.

Reply:
It is still used in `FillPerChannelMinMax` https://github.com/tensorflow/tensorflow/pull/51527/files#diff-26ec535332e9824d3089caee17d4c466a758ba7f5f3a5b4fa8a6ed11e3edd243R138